apollo-router/src/spec/field_type.rs

```diff
@@ -112,7 +112,18 @@ fn validate_input_value(
     match type_name.as_str() {
         "String" => return from_bool(value.is_string()),
         // Spec: https://spec.graphql.org/June2018/#sec-Int
-        "Int" => return from_bool(value.is_valid_int_input()),
+        "Int" => {
+            let valid = value.is_valid_int_input();
```

Comment:
Can this validation return any other error that is not the larger-than-32bit issue?

Reply:
I felt like it was weird to put this here where I did. However, no it just does `self.as_i64().is_some() || self.as_u64().and_then(|x| i64::try_from(x).ok()).is_some()` and it felt weird to put then logging in the `is_valid_int_input` because that just determines if it's valid or not and doesn't take any action based on that (could maybe be called from elsewhere).

Reply:
I mean, can the warning in the lines below happen for errors that are not of this type?